extern/sector-storage/sched.go

```diff
@@ -204,9 +204,14 @@ type SchedDiagRequestInfo struct {
 	Priority int
 }
 
+type SchedDiagWorkerInfo struct {
+	WorkerID
+	AcceptedTaskTypes map[sealtasks.TaskType]struct{}
```

Comment:
Could just be `[]sealtasks.TaskType`

Reply:
(bonus points for having it sorted)

Reply:
I did that originally, but this means I don't have to transform the output of `tt, err := sh.workers[window.worker].w.TaskTypes(context.TODO())` Happy to change it back